### DISCUSSION TITLE
Establishing and Supporting a Vibrant Internal R Community

### DISCUSSION THREAD

**Discussion by vlprince** [rinpharma/rinpharma-summit-2024#9] [roundtable]
### Proposal

This discussion will invite participants to explore how we can initiate and cultivate an R community, especially for companies that are just entering the open-source world. We're **keen on learning from the pros**—the trailblazers who've been there and done that—to soak up all their hard-earned wisdom. But it's just as important to connect with those of us, who are **navigating the initial stages of adoption**, to hear about the hurdles they're facing and their discoveries along the way. It's a chance for everyone, no matter where they're at in their open-source journey, to trade tips and share stories.

### Expected impact

By sharing experiences about what's worked and what hasn't, we can dodge the usual traps. Plus, this discussion can provide practical tips to keep our R communities thriving and adapting to the evolving data science landscape in pharma. It's all about setting ourselves up for success down the road.

### Prior discussions/work

There have been presentations on this topic during prior conferences:
[Successfully building a vibrant community of R users](https://www.youtube.com/watch?v=88mChWzxKwQ&ab_channel=RinPharma)
[Creating a Centre of Excellence for R in a Large Pharma Company](https://www.youtube.com/watch?v=F6t1miqZMZM&ab_channel=SiliconBrighton)

### Would you be willing to potentially facilitate this discussion?

Yes definately!

**Comment by telepath37** [comment]
Happy to share our experience (setting up internal R/Shiny competition, building R Community, developing R packages, open-sourcing, filing with R, etc.). The success of R adoption might need a mix of factors (individual motivation, management support, infrastructure, etc.).

**Reply by vlprince** [reply]
@telepath37 That would be so great to hear about your successes and lessons learned!
